### ISSUE TITLE
Party claiming mode

### ISSUE THREAD

**Issue by Pannoniae** [thexaero/open-parties-and-claims#166]
There could be an option so claims are associated with parties, not people.
Context: I am making a roleplay kingdoms server. People could use parties to live in one kingdom, sure. But when someone leaves, obviously they continue to claim the same area which they have claimed when they were in the party. They should just lose access to the claimed area and the party leader should control who has access.

**Comment by thexaero** [comment]
I'll consider it.
I definitely would not prefer playing like this. This means that you lose your claimed builds when you leave a party and the party keeps all the extra claims that might go beyond the limit, if the limit is based on the party size.

**Comment by Pannoniae** [comment]
Fair enough, it does suck as a player. However the current method sucks for the party (and the whole server itself).
If you leave the party the admin needs to intervene to manually unclaim the chunks from the player who left.

This is mainly an issue for things like common town buildings, etc. which do not really have a singular owner in the party but *someone* must claim it.

An alternate solution right now could be to have claims be owned not by the party but the owner of the team I guess and disallow everyone else from claiming. I will probably do that (set claim limit to 0 and manually increase it for the kingdom leaders)

**Comment by thexaero** [comment]
There should probably still be a way to claim as yourself and not the party. I'll have to think about a good way to implement this, if I decide to do it.

**Comment by Pannoniae** [comment]
Alright:)

**Comment by thexaero** [comment]
After discussing this with another user, I'm thinking about adding options to give party moderators/admins the ability to claim as the party leader. Could also display the party itself as the claim owner on the map instead of the party owner.

**Comment by OzTheM9** [comment]
I would really like to see a party claiming mode as we have a lot of players in teams. Forceloads also only work if the person who claimed the chunks is online. Having a sum of claim chunks from every player in the team seems like a good idea, maybe with limits like FTBChunks does it. 

Our players seem very confused by the way claiming/loading chunks as a team and with allies works, so a party claim mode would be best.

https://thexaero.github.io/open-parties-and-claims/javadoc/xaero/pac/common/server/parties/system/api/IPlayerPartySystemAPI.html#isPermittedToPartyClaim(java.util.UUID)

It seems like there's already methods for this, but they're unused, so I'm not sure what the progress is.

**Comment by nichism** [comment]
Looking for this same feature
